packages/devp2p/src/protocol/protocol.ts

```diff
@@ -72,7 +71,7 @@ export class Protocol extends EventEmitter {
 
     // Remote Peer IP logger
     const ip = this._peer._socket.remoteAddress
-    if (isTruthy(ip)) {
+    if (ip !== undefined) {
```

Comment:
Same as above, would it make more sense to explicitly check for `typeof ip === 'string'` or is that just ugly?

Reply:
Done